docs/source/en/model_doc/git.mdx

```diff
@@ -73,4 +73,9 @@ The resource should ideally demonstrate something new instead of duplicating an
 ## GitForCausalLM
 
 [[autodoc]] GitForCausalLM
-    - forward
\ No newline at end of file
+    - forward
+
+## GitForVisualQuestionAnswering
```

Comment:
Hi @marechaux thanks a lot for working on this!

However we don't need to add a new class for it actually, as GitForCausalLM handles VQA as shown in t[his code snippet](https://huggingface.co/docs/transformers/model_doc/git#transformers.GitForCausalLM.forward.example-2)

Reply:
Thanks, I just rewrite it without adding a new model